src/sql/mir.rs

```diff
@@ -785,6 +808,142 @@ impl SqlToMirConverter {
         )
     }
 
+    fn make_predicate_nodes(
+        &self,
+        name: &str,
+        parent: MirNodeRef,
+        ce: &ConditionExpression,
+        nc: usize,
```

Comment:
Yeah, it's supposed to be the count of nodes inside the predicate. We had something similar for filters, so I kept it. I don't think there's any reason why this couldn't be the total node count instead.